IntegrationTests/ImageCachePolicyTest.js

```diff
@@ -65,7 +65,7 @@ class ImageCachePolicyTest extends React.Component<Props, $FlowFixMeState> {
         <Text>Hello</Text>
       <Image
         source={{
-              uri: 'https://facebook.github.io/react/img/logo_small_2x.png?cacheBust=notinCache' + Date.now(),
+              uri: 'https://facebook.github.io/react-native/img/logo-small-2x.png?cacheBust=notinCache' + Date.now(),
```

Comment:
This image is not loading for me, either.

Reply:
These URLs won't work until are added to the website page (after merged the PR)

Reply:
If this were a simple website change, it would be alright as the website is versioned and the references to the new images will only go live once the images are deployed.

In this case, I think the images won't be visible on https://facebook.github.io/react-native/ for up to two months due to the way the website is deployed.

Let's first use existing images in the repo itself. Maybe update existing images instead, if that does not break anything unrelated?

Reply:
We could use this URL raw.githubusercontent.com/*gusgard*/react-native/*correct-url-logo*/website/src/react-native/img/logo-small.png that don't depend on the website. (changing user and branch for facebook and master). 

Otherwise, I could use the images that are already in the img folder and test it well.